### ISSUE TITLE
Unable to start Cloudflare workers project with capital letters in directory name

### ISSUE THREAD

**Issue by nmemoto** [honojs/create-hono#13]
This may be a minor issue, but I'll share.

I ran the following

```
$ npm create hono@latest todoAPI
Need to install the following packages:
create-hono@0.3.1
Ok to proceed? (y) y

create-hono version 0.3.1
✔ Using target directory … todoAPI
✔ Which template do you want to use? › cloudflare-workers
cloned honojs/starter#main to /workspaces/*****/todoAPI
✔ Copied project files
```

After `cd todoAPI && npm i` , Run the following

```
$ npm run dev

> dev
> wrangler dev src/index.ts

 ⛅️ wrangler 3.14.0
-------------------

✘ [ERROR] Processing wrangler.toml configuration:

    - Expected "name" to be of type string, alphanumeric and lowercase with dashes only but got
  "todoAPI".
```

The wrangler.toml generated at this time is as follows

```toml:wrangler.toml 
name = "todoAPI"
compatibility_date = "2023-01-01"

.......
```

I thought it would be better to kebab case and convert all to lower case when replacing by project name from the template.
https://github.com/honojs/create-hono/blob/6aa68a3be01d4e1ed5034b5701ce83017e7f731e/src/hooks/after-create.ts#L12

However, I can't use this method when the project name is a non-ascii character or a non-hyphen symbol. Therefore, I think it would be better to dare to do nothing.

**Comment by yusukebe** [comment]
@nmemoto 

Thank you for raising the issue. This is good point.

@sor4chi any thought?

**Comment by sor4chi** [comment]
Hi, @nmemoto  @yusukebe

It is true, if you ask me.
I have an image that many Create CLI converts App name to snake-case and appropriates it to folder name or app name.

How do you think about this plan?

**Comment by nmemoto** [comment]
@sor4chi @yusukebe Thanks for confirming.
How about following wrangler?
It looks like the value of name in wrangler.toml that wrangeler generate creates is set [here](https://github.com/cloudflare/workers-sdk/blob/d74c35fc608f650fe82c12e62dfde47956a4db43/packages/wrangler/src/init.ts#L285).

The workerName is set [here](https://github.com/cloudflare/workers-sdk/blob/d74c35fc608f650fe82c12e62dfde47956a4db43/packages/wrangler/src/init.ts#L146-L149).
Looking at this process, we see that it converts to lowercase and also replaces all invalid characters with hyphens.

**Comment by yusukebe** [comment]
Hi @nmemoto !

I've released the v0.3.2 which is including your fix. Great work! Thanks.
